src/main/resources/static/css/style.css

```diff
@@ -0,0 +1,505 @@
+        border-radius: 0px 10px 3px 0px;
+        background:rgba(0, 0, 0, 1);
+    }
+
+    #mobile_nav_buttons_right{
+
+        position: absolute;
+
+        top: 5px;
+        right: 0px;
+
+        display: flex;
+        justify-content: flex-end;
+        align-content: center;
+        align-items: center;
+
+        width: calc(100vw - 70px);
+        height: 50px;
+        padding-right: 20px;
+        border-radius: 10px 10px 0px 0px;
+        background:rgba(0, 0, 0, 1);
+    }
+
+        h1{
+        font-size: 2vh;
+    }
+
+    #logo_frame {
+        height: 3vh;
+        width: 3vh;
+        background-size: 2.5vh 2.5vh;
+
+    }
+
+    #report_container{
+        width: 100%;
+        position: relative;
+    }
+
+    #o80696 {
+        width: 100%;
+    }
+
+    #add_tag{
+        position: absolute;
+        bottom: 10px;
+        left:10px;
+    }
+
+    #map{
+        width: 100%;
+        height: 300px;
+    }
+
+    form{
+         width: 50%;
+    }
+
+}
+
+@media screen and (orientation: landscape) {
+
+    * {
+
+    }
+
+    body{
+        position: absolute;
+        top: 5vh;
+    }
+
+    head_space {
+        height: 5vh;
+        }
+
+    div > nav > div > a{
+        display: none;
+    }
+
+    header {
+        height: 5vh;
+    }
+
+    #logo_view{
+        float:left;
+        margin-left: 5px;
+        display: flex;
+        align-items: center;
+    }
+
+    body > header > nav{
+        display: inline-block;
+        position: fixed;
+        right: 0px;
+        margin-right: 10px;
+    }
+
+    #nav_m{
+        display:none;
+    }
+
+    #teams{
+        display:flex;
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        width: 100%;
+    }
+
+    .team_card{
+        padding: 5px;
+        width: 19vw;
+        min-width: 200px;
+        height: 150px;
+        background:rgba(0, 0, 0, 0.7);
+        display: inline-block;
+    }
+
+    #create_team{
+        position: relative;
+        top: 370px;
+    }
+
+    .shades{
+
+        background:rgba(0, 0, 0, 0.7);
+        padding: 5px 5px;
+        border-radius: 3px;
+        margin-left: 5px;
+        margin-top: 5px;
+    }
+
+    #mobile_nav{
+        display: none;
+    }
+
+    #mobile_nav_buttons_right{
+
+        position: absolute;
+
+        top: 0px;
+        right: 10px;
+
+        display: flex;
+        justify-content: flex-end;
+        align-content: center;
+        align-items: center;
+
+        width: calc(100vw - 70px);
+        height: 5vh;
+        margin: 0px 0px 0px 0px;
+
+
+    }
+
+    .team_card{
+        background:rgba(0, 0, 0, 0.8);
+        padding: 5px 5px;
+        border-radius: 5px;
+        margin-left: 5px;
+        margin-top: 5px;
+}
+
+    #vertical_locations{
+        display:inline-block;
+        width: 200px;
+        height: 365px;
+        float: left;
+        font-family: 'Viga', sans-serif;
+        overflow: auto;
+    }
+
+    .resort_button_div{
```

Comment:
You set the margin of this class twice, in two different declarations. Your CSS could be *much* cleaner.